charts/repo-proxy/templates/hpa.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
```

Comment:
Is there a reason to use an HPA for this service?

Reply:
None, I've used `helm create repo-proxy` to generate the chart, this was included.